mkdocs.yml

```
@@ -247,6 +247,8 @@ nav:
               - tests/integration/test_fim/test_files/test_max_eps/index.md
               - Test max eps synchronization:  tests/integration/test_fim/test_files/test_max_eps/test_max_eps_synchronization.md
               - Test max eps: tests/integration/test_fim/test_files/test_max_eps/test_max_eps.md
+            - Test max files per second:
+              - test max files per second: tests/integration/test_fim/test_files/test_max_files_per_second/test_max_files_per_second.md
```

```suggestion
              - tests/integration/test_fim/test_files/test_max_files_per_second/test_max_files_per_second.md
```
If there's only a test in this directory, you can just list its markdown.